src/loaders/compute_tools/tool_common.py

```diff
@@ -361,11 +361,13 @@ def _create_metadata_file(
     # create tool genome identifier metadata file
     genome_meta_file_path = os.path.join(batch_dir, loader_common_names.GENOME_METADATA_FILE)
     with open(genome_meta_file_path, "w") as meta_file:
+        meta_file.write(f"{_META_TOOL_IDENTIFIER}\t{_DATA_ID_COLUMN_HEADER}\t{_META_SOURCE_FILE}\t"
+                        + f"{_META_SOURCE_DIR}\t{_META_SOURCE_FILE_NAME}\n")
         for genome_id, genome_meta_info in meta.items():
             meta_file.write(
                 f'{genome_meta_info[_META_TOOL_IDENTIFIER]}\t{genome_id}\t'
-                + f'{genome_meta_info[_META_SOURCE_FILE]}\t'
                 + f'{genome_meta_info[_META_SOURCE_DIR]}\t'
+                + f'{genome_meta_info[_META_SOURCE_FILE]}\t'
```

Comment:
`_META_SOURCE_FILE` should come before `_META_SOURCE_DIR` to match the header.

Reply:
I didn't catch it from previous PR. But I think `_META_SOURCE_FILE_NAME` should be `_META_FILE_NAME`.

Reply:
I asked Sijie to put the directory before the files

Reply:
Updated!